### DISCUSSION TITLE
[question] Is there any way to get rid of the <?xml version="1.0" encoding="UTF-8"?> string from the XML file?

### DISCUSSION THREAD

**Discussion by wujekbogdan** [downforcetech/fluidxml.php#33] [Questions]
I have to generate a very big XML file. The file is generated incrementally - node by node, then these nodes are concatenated into a final XML file (partials are saved on a disc and then merged when all partials are ready).

I don't want each node to contains the `<?xml version="1.0" encoding="UTF-8"?>` so I have to use `str_replace()` in order to get rid of this tag from each node. Is there a better, more elegant way?

**Comment by daniele-orlando** [comment]
Hi dear.

https://github.com/servo-php/fluidxml/wiki/Getting-Started:-10.-Exporting

> ```php
> $doc->xml(true);
> ```
> ```xml
> <book>
> ...
> </book>
> ```

**Comment by wujekbogdan** [comment]
Ups... Thank you :).
